go.mod

```diff
@@ -0,0 +1,34 @@
+module github.com/terraform-providers/terraform-provider-linode
+
+require (
+	github.com/apparentlymart/go-cidr v1.0.0 // indirect
+	github.com/armon/go-radix v1.0.0 // indirect
+	github.com/aws/aws-sdk-go v1.16.27 // indirect
+	github.com/blang/semver v3.5.1+incompatible // indirect
+	github.com/dnaeon/go-vcr v1.0.1 // indirect
+	github.com/hashicorp/go-getter v1.0.2 // indirect
+	github.com/hashicorp/go-hclog v0.0.0-20190109152822-4783caec6f2e // indirect
+	github.com/hashicorp/go-plugin v0.0.0-20190129155509-362c99b11937 // indirect
+	github.com/hashicorp/go-uuid v1.0.1 // indirect
+	github.com/hashicorp/hcl v1.0.0 // indirect
+	github.com/hashicorp/hcl2 v0.0.0-20190130225218-89dbc5eb3d9e // indirect
+	github.com/hashicorp/hil v0.0.0-20190129155652-59d7c1fee952 // indirect
+	github.com/hashicorp/logutils v1.0.0 // indirect
+	github.com/hashicorp/terraform v0.11.11
```

Comment:
We have a beta version of v0.11.12 that is switched to go modules. We are recommending upgrading to it to clean up all the indirect dependencies. It's not necessary but recommended before upgrading to v0.12 (which is on modules).

This beta release has a named branch you can use for go get
```
go get github.com/hashicorp/terraform@sdk-v0.11-with-go-modules
```

Reply:
you should then run
```
go mod tidy
go mod vendor
```